return:
  success
output-includes:
  - "did not find foo"
